plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestTimestamp.java

```diff
@@ -71,29 +79,36 @@ public void testTimestampBackedByInt64()
             MessageType parquetSchema = parseMessageType("message hive_timestamp { optional int64 test (" + logicalAnnotation + "); }");
 
             Iterable<Long> writeNullableDictionaryValues = limit(cycle(asList(1L, null, 3L, 5L, null, null, null, 7L, 11L, null, 13L, 17L)), 30_000);
-            testRoundTrip(parquetSchema, writeNullableDictionaryValues, timestamp);
+            testRoundTrip(parquetSchema, writeNullableDictionaryValues, timestamp, dateTimeZone);
 
             Iterable<Long> writeDictionaryValues = limit(cycle(asList(1L, 3L, 5L, 7L, 11L, 13L, 17L)), 30_000);
```

Comment:
Why aren't we using also values which are closer the current times (not only very close to 1970-01-01) for testing?

Reply:
`AbstractTestParquetReader#testTimestamp` simulates round trip with current timestamp.